### PR TITLE
Space Travelers' Hub

### PR DESCRIPTION
**Pull request by awais-amjed**
Space Travelers' Hub is a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and dragons and join selected space missions. It is built in react and redux by using SpaceX API.

Link to Live demo: [demo](https://team-space-travelers.netlify.app/)

Screenshot

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <h1>Space Travelers&apos; Hub</h1>
+      </div>
+      <nav>
+        {links.map((link) => (
+          <NavLink
+            key={link.id}
+            to={link.path}
+          >
+            {link.text}
+          </NavLink>
```
Comment (micheaol) [inline src/components/Header.js:42]: Please ensure that the active page should be indicated with an underline according to the project requirement.

Your Implementation | Figma Design
:---:|:---:
image | image

File: package.json [hunk 2]
```diff
@@ -3,12 +3,22 @@
...
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.1.3",
+    "node-sass": "^7.0.1",
+    "prop-types": "^15.8.1",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.2.3",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.8",
```
Comment (micheaol) [inline package.json:16]: Please ensure that your `package.json` includes `Redux Logger` according to the project requirement.

File: src/pages/MyProfilePage.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './MyProfilePage.scss';
+
+function MyProfilePage() {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions);
+  const dragonsList = useSelector((state) => state.dragons);
+  const filteredDragons = dragonsList.filter((dragon) => dragon.dragonReserved);
```
Comment (micheaol) [inline src/pages/MyProfilePage.js:10]: Please ensure that you render `joined missions` and `reserved rockets` using `filter();` just as you have done with `joined dragon` according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Header.js (hunk 1 above)
Comment (awais-amjed) [reply src/components/Header.js:42]: Thanks for the comments. The changes have been applied. ✅

File: src/pages/MyProfilePage.js (hunk 3 above)
Comment (awais-amjed) [reply src/pages/MyProfilePage.js:10]: Thanks for the comments. The changes have been applied. ✅

File: package.json (hunk 2 above)
Comment (awais-amjed) [reply package.json:16]: Thanks for the comments. The changes have been applied. ✅

**Review by Emmyn5600 — APPROVED** [review]
Hi @awais-amjed @M0rrighan @selma-belhadj 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**